### PR TITLE
The actual church buff PR (ft. Bath salts)

### PR DESCRIPTION
**Pull request by Lamasmaster**
## About The Pull Request

Adds new church themed artifacts to prime's office (chalice that generates cahors or urn with ash).

Renames cruciform in status tab to Axis Power.

Fixes carpediem somehow being onehanded weapon.

Adds sprites without pauldrons to the toga as alts and fixes the church cloaks not showing on suit slot.

Gives the Harmony Scepter and Carpediem their own unique firing sounds.

Adds an oddity to foreman's office

N
N2

## Changelog
:cl:
add: Church Artifacts
add: Peggy
add: Toga alts
tweak: crucifom to Axis Power
balance: Harmony Staff
fix: Church cloaks
fix: Carpediem
/:cl:

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
Hold on here

File: code/game/objects/items/oddities.dm [hunk 1]
```diff
@@ -520,6 +520,19 @@
 	)
 	perk = /datum/perk/guild/blackbox_insight
 
+/obj/item/oddity/bath //Oink
+	name = "Peggy's Bathwater"
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/oddities.dm:524]: No just no

File: code/game/objects/items/weapons/autolathe_disk/absolute_disks.dm [hunk 2]
```diff
@@ -615,6 +615,7 @@ obj/item/computer_hardware/hard_drive/portable/design/nt/concillium
 		/datum/design/autolathe/shield/nt_shield,
 		/datum/design/autolathe/nt/shield/nt_buckler,
 		/datum/design/autolathe/gun/plasma/martyr,
+		/datum/design/bioprinter/healingstaff,
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/autolathe_disk/absolute_disks.dm:618]: No. Hard no its prime only

File: code/datums/autolathe/biomatter.dm [hunk 3]
```diff
@@ -446,6 +446,11 @@
 	name = "\"Tersus\" Vacuum Cleaner"
 	build_path = /obj/item/holyvacuum
 
+/datum/design/bioprinter/healingstaff
```
Comment (Trilbyspaceclone) [inline code/datums/autolathe/biomatter.dm:449]: This is prime only do not add it to any disks, its hand made

File: code/game/objects/items/weapons/candle.dm [hunk 4]
```diff
@@ -71,3 +71,19 @@
 		lit = 0
 		update_icon()
 		set_light(0)
+
+/obj/item/flame/candle/eternal
+	name = "eternal candle"
+	desc = "A curved, comma-shaped jewel made out of precious stone. Simply looking at this stone is enough to make one more calm and confident. It is said Amaterasu herself - one of the absolute's masks - held this specific Magatama herself, and anyone that holds this jewel while meditating long enough, finds inner peace easier than in any other way.  In fact, during times of turmoil it became well know to allow vectors to touch the Magatama before sleeping on monasteries, due to it's non-chemically related, calming properties. It is also said this Jewel was totally unrepaired by the church, and because of time, shows some external damages and scratches."
+	icon_state = "candle_eternal"
+	light_color = COLOR_LIGHTING_CYAN_BRIGHT
+	endless_burn = TRUE
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/candle.dm:80]: Endless burn dosnt make wax go down.

File: code/modules/projectiles/projectile/beams.dm [hunk 5]
```diff
@@ -203,17 +203,23 @@ In pvp they also have more lasting damages, such as infections, and pain form bu
...
+	stun = 0
+	weaken = 0
+	eyeblur = 0
+	muzzle_type = /obj/effect/projectile/laser_blue/muzzle
+	tracer_type = /obj/effect/projectile/laser_blue/tracer
+	impact_type = /obj/effect/projectile/laser_blue/impact
 
-/obj/item/projectile/beam/sniper/healing/staff/on_hit(atom/target, blocked = FALSE)
-	. = ..()
+/obj/item/projectile/beam/sniper/harmony/on_hit(atom/target, blocked = FALSE)
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/projectile/beams.dm:218]: Dont  make it inject chemicals. The way it was was done in a direct way for a direct reason

File: code/modules/psionics/psionic_items/psi_misc.dm [hunk 6]
```diff
@@ -110,3 +110,11 @@
...
             return TRUE
     return FALSE
+
+/obj/item/clothing/gloves/psionic_ring
+	name = "Ring of Dispel"
+	desc = "A ring belonging from the old poems of Le Chevalier de la Charrette, given to Sir Lancelot by the Lady of the Lake. This ring is, supposedly, able to dispel any enchantment, but one can't say for sure considering the story of this ring to be false. Whatever the reason, numericals theorize this ring may have come from Chrétien de Troyes (the writer of the poem) himself and noticed the curious, anomalitic nature was a good reference to write a book about.\
+			The ring of Dispel can be used during Psion crisis but the vector or prime using said ring must be safeguarded by other vectors, considering said vector as a artifact as much as the ring is until it is put back to it's place. Holding the ring of dispel is a chore because it basically doubles the expectation for the vector to not die, or lose it."
+	icon_state = "ring_of_dispel"
+	price_tag = 1000
+	psi_blocking = 25
```
Comment (Trilbyspaceclone) [inline code/modules/psionics/psionic_items/psi_misc.dm:120]:
```suggestion
	psi_blocking = 50
```

**Comment by Trilbyspaceclone** [comment]
Show screen shots of what you mapped

File: code/game/objects/items/weapons/candle.dm (hunk 4 above)
Comment (Trilbyspaceclone) [reply code/game/objects/items/weapons/candle.dm:80]: NVM this has no melting states is all good

File: code/game/objects/items/oddities.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/game/objects/items/oddities.dm:524]: Make it just browns blessed holy water
or something not stupid

**Comment by Lamasmaster** [comment]
All the sprites were made by Polyushko#0323 but since I am stupid and forgot to put it into the description I will at least put it here

**Pull request merged**